import_export/tmp_storages.py

```diff
@@ -71,7 +71,7 @@ class MediaStorage(BaseStorage):
     MEDIA_FOLDER = "django-import-export"
 
     def __init__(self, name=None, read_mode="rb", encoding=None):
-        super().__init__(name, read_mode=read_mode, encoding=encoding)
+        super().__init__(name, read_mode="rb", encoding=None)
```

Comment:
I don't understand why we switch from using the supplied read_mode variable to hardcoded "rb" here - feels like a recipe for confusion, though I don't know what the use case is for supplying a different read mode. Maybe we can't fix the method signature easily?

Reply:
I agree it looks clunky.  The reason is that the call from [admin.py](https://github.com/django-import-export/django-import-export/blob/432c5351c42f319fe98aa3ed22170a8726b5e5d4/import_export/admin.py#L478) always passes these parameters, but `read_mode` is unnecessary here because we always want to read in binary (this is the nature of the error in #1589).

I changed the `tmp_storages.py` implementation to allow for optional kwargs.